src/app/policies/components/PolicyConnections.spec.ts

```diff
@@ -100,7 +100,7 @@ describe('PolicyConnections.vue', () => {
   test('renders no item', async () => {
     const server = useServer()
     server.use(
-      rest.get(import.meta.env.VITE_KUMA_API_SERVER_URL + '/meshes/:mesh/:policyPath/:policyName/dataplanes', (req, res, ctx) =>
+      rest.get(import.meta.env.VITE_KUMA_API_SERVER_URL + '/meshes/:mesh/:policyPath/:policyName/dataplanes', (_req, res, ctx) =>
```

Comment:
nitpick: _req seems not used so why the rename?

```suggestion
      rest.get(import.meta.env.VITE_KUMA_API_SERVER_URL + '/meshes/:mesh/:policyPath/:policyName/dataplanes', (req, res, ctx) =>
```

Reply:
Yeah, I now realize we should leave this as `req` to trigger the linter error for unused variables should this be refactored to no longer access the other positional arguments. I’ll change it back in a follow up.